### PR TITLE
Revamp messages list layouts

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

File: client/app/styles/_message-list.styl [hunk 1]
```diff
@@ -1,36 +1,40 @@
...
-.messages-list .preview
-    max-width 30vw
-
-.full .messages-list .preview
-    max-width 85vw
+// .messages-list .preview
+//     max-width 30vw
+//
+// .full .messages-list .preview
+//     max-width 85vw
```
Comment (nono) [inline client/app/styles/_message-list.styl]: @m4dz is it normal to have some code commented? Is it dead code?

File: client/app/styles/_message-list.styl [hunk 2]
```diff
@@ -175,227 +179,276 @@
...
+    //         .subject
+    //             color #666
+    //
+    //         .participants
+    //             font-weight normal
+    //
+    //
+    //     // Currently selected message
+    //     main:not(.full) &.active
+    //         background-color #e5e5e5
```
Comment (nono) [inline client/app/styles/_message-list.styl]: @m4dz same here

File: client/app/styles/_message-list.styl (hunk 1 above)
Comment (m4dz) [inline client/app/styles/_message-list.styl]: Oops, clean up fail :smile:. Thanks!

**Comment by poupotte** [comment]
Functionally, it's ok for me :) !

**Comment by poupotte** [comment]
@m4dz , @nono , Can we merge this PR ?

**Comment by m4dz** [comment]
@nono : yep, probably. I'll work on the build process in a week :wink: (and I'll probable use `prebuild` for cleaning step)

@poupotte : ok for me :+1:

**Comment by frankrousseau** [comment]
Thank you all for your participation @m4dz @poupotte @nono.

**Pull request merged**